src/main/java/com/redhat/coolstore/model/InventoryEntity.java

```diff
@@ -1,74 +1,20 @@
 package com.redhat.coolstore.model;
 
-import java.io.Serializable;
-
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.Table;
-import javax.persistence.UniqueConstraint;
-import javax.xml.bind.annotation.XmlRootElement;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
 
 @Entity
-@XmlRootElement
-@Table(name = "INVENTORY", uniqueConstraints = @UniqueConstraint(columnNames = "itemId"))
-public class InventoryEntity implements Serializable {
-
-	private static final long serialVersionUID = 7526472295622776147L; 
+public class InventoryEntity extends PanacheEntityBase {
 
     @Id
-    private String itemId;
-
-
-    @Column
-    private String location;
-
-
-    @Column
-    private int quantity;
-
-
-    @Column
-    private String link;
-
-    public InventoryEntity() {
-
-    }
-
-    public String getItemId() {
-		return itemId;
-	}
-
-	public void setItemId(String itemId) {
-		this.itemId = itemId;
-	}
-
-	public String getLocation() {
-		return location;
-	}
-
-	public void setLocation(String location) {
-		this.location = location;
-	}
-
-	public int getQuantity() {
-		return quantity;
-	}
-
-	public void setQuantity(int quantity) {
-		this.quantity = quantity;
-	}
```

Comment:
This `PanacheEntityBase` change is something that I need to investigate, it may be correct and it's just being smart, but it's something that we don't have encoded in our rules.